tools/scripts/gen-env-files.sh

```diff
@@ -444,15 +444,15 @@ if test -f "./services/audio/audio.env"; then
     echo "./services/audio/atom.env exists"
 else
 echo \
-"KEYCLOAK_AUTH_SERVER_URL=http://host.docker.internal:40001/auth/
+"KEYCLOAK_AUTH_SERVER_URL=http://host.docker.internal:$portKeycloak/auth/
 
-DB_URL=jdbc:postgresql://host.docker.internal:40000/tno
-DB_USERNAME=stuartm
-DB_PASSWORD=YTc0YjE3ZGU3N2Q4OTkyZDc3OWEwNA8!
```

Comment:
Don't save passwords in source.  This one is just a generated temporary local one, but still.